api/src/components/flag/controller.ts

```diff
@@ -53,13 +53,23 @@ export const createFlag = async (
     try {
         const publication = await publicationService.get(event.pathParameters.id);
 
-        if (!publication || publication.currentStatus !== 'LIVE') {
+        if (!publication || !publication.versions.some((version) => version.currentStatus === 'LIVE')) {
```

Comment:
Since flags can only be raised against the latest published version, this condition needs to check whether the latest LIVE version of this publication exists.